<launch>
  <node pkg="ros_awsiot_agent" type="ros2mqtt.py" name="ros2mqtt" output="screen" >
    <rosparam command='load' file="$(find ros_awsiot_agent)/param/conn_param.yaml"/>
    <param name="~topic_from" value="/diagnostics_agg"/>
    <param name="~topic_to" value="/ros"/>
  </node>
  <node pkg="ros_awsiot_agent" type="mqtt2ros.py" name="mqtt2ros" output="screen" >
    <rosparam command='load' file="$(find ros_awsiot_agent)/param/conn_param.yaml"/>
    <param name="~topic_from" value="/ros"/>
    <param name="~topic_type" value="diagnostic_msgs/DiagnosticArray"/>
  </node>
</launch>